Lib/typing.py

```diff
@@ -591,12 +591,6 @@ def __repr__(self):
 # * __args__ is a tuple of all arguments used in subscripting,
 #   e.g., Dict[T, int].__args__ == (T, int).
 
-_sentinel = object()
-
-
-def _is_dunder(attr):
```

Comment:
Why this method has been removed? I think it adds readability.

Reply:
I typically use the rule of three times: "If something is used three times, then factor it out". Here it is used only twice. But if you like it, then I can put it back. Please confirm.

I agree with all other comments.

Reply:
Use what looks better to you.

Reply:
OK, I restored the function (it is easier than wrapping those lines).